integration/setup/provider.go

```diff
@@ -22,12 +22,6 @@ func provider(ctx context.Context, config Config) error {
 			},
 			Secret: chartvalues.AzureOperatorConfigSecret{
 				AzureOperator: chartvalues.AzureOperatorConfigSecretAzureOperator{
-					CredentialDefault: chartvalues.AzureOperatorConfigSecretAzureOperatorCredentialDefault{
```

Comment:
Is this fine to be just deleted? That field doesn't exist in upstream library any more and tests pass with this.

Reply:
It got deleted but I am not sure if something needs to be added to the e2e test then. Pawel worked on that for AWS.

Reply:
See his changes here:
https://github.com/giantswarm/aws-operator/pull/1880/files

Reply:
Ok. This got fixed here: https://github.com/giantswarm/azure-operator/pull/559/files